renderer/components/ActivitiesTable.tsx

```diff
@@ -407,6 +407,7 @@ export default function ActivitiesTable({
                     className={clsx({
                       "py-1 px-2 -mx-2 rounded-full font-medium bg-yellow-100 text-yellow-800 dark:text-yellow-400 dark:bg-yellow-400/20":
                         activity.mistakes?.includes("startsWith!"),
+                      "old-break-word": true,
```

Comment:
You can write without `true` like this 
`className={clsx("old-break-word", {
                      "py-1 px-2 -mx-2 rounded-full font-medium bg-yellow-100 text-yellow-800 dark:text-yellow-400 dark:bg-yellow-400/20":
                        activity.mistakes?.includes("startsWith!")
                    })}`